server/src/main/java/jass/server/entity/TeamEntity.java

```diff
@@ -68,6 +82,14 @@ public int getId() {
      * @return Returns player one.
      */
     public UserEntity getPlayerOne() {
+        if (!playerOneLoaded) {
+            try {
```

Comment:
instead of all this I would add foreignAutoRefresh = true to the field attribute